Assets/Scripts/Blobs/BlobStats.cs

```diff
@@ -6,15 +6,19 @@ namespace Blobs
 {
     public class BlobStats : MonoBehaviour
     {
-        [Tooltip("This should be kept to a small amount as this is percent increase")][SerializeField]
-        private float blobScore=10;
+        [Tooltip("This should be kept to a small amount as this is percent increase")] [SerializeField]
+        private float blobScore = 10;
+
+        //Sends score increase if the player collides with the blob
         private void OnTriggerEnter(Collider other)
         {
             //TODO: implement score increase
```

Comment:
I see a send blob score event, if this is related can remove the comment.

Reply:
what?

Reply:
You should remove the TODO comment if you already implemented the score increase. At least from looking at the event below for SendBlobScore, it looks like you did.

Reply:
Ohhh its kinda hard to see what line exactly you comment on.

Reply:
Fixed